static/scss/search-page.scss

```diff
@@ -365,3 +365,20 @@
     color: rgba(0,0,0,0.8);
   }
 }
+
+.sk-hierarchical-menu-list {
+  width: 100%;
+
+  .sk-hierarchical-menu-list__root {
+    margin-left: 10px;
+    margin-right: 10px !important;
```

Comment:
@singingwolfboy I am going to venture to guess that a lot of the times we are using !important are not really necessary. I am not sure about this particular case. But in general this should be avoided if possible. In the previous case, it was overriding inline styles, which I think, makes using !important correct. But in may cases, it is just overriding a more specific css declaration, and in these cases using !important is bad practice. My concern with the idea of "adding a comment whenever we use !important" is that the implication is not just add a comment, but give a good reason for using it. What I am getting at is that the implication is to reassess the use of !important, which is a larger issue.